src/GraphQL/StringExtensions.cs

```diff
@@ -64,5 +64,47 @@ private static string FastChangeFirstLetter(char newFirstLetter, string s)
             s.AsSpan().Slice(1).CopyTo(buffer.Slice(1));
             return buffer.ToString();
         }
+
+        /// <summary>
+        /// Returns a constant case version of this string. For example, converts 'StringError' into 'STRING_ERROR'.
+        /// </summary>
+        public static string ToConstantCase(this string value)
+        {
+            //aka: return Regex.Replace(s, @"([A-Z])([A-Z][a-z])|([a-z0-9])([A-Z])", "$1$3_$2$4").ToUpperInvariant();
+            int i;
+            int strLength = value.Length;
+            // iterate through each character in the string, stopping a character short of the end
```

Comment:
_stopping a character short of the end_ ?

Reply:
```
Offset: 0123456
String: TEST
```
Because the iterator needs to always check the current and following character, it scans from index 0 to index 2.

If the comment is unclear please update it as you see fit.